frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -58,6 +58,15 @@ export const UninstallOperatorModal = withHandlePromise((props: UninstallOperato
       .catch(_.noop);
   };
 
+  const uninstallMsg =
```

Comment:
I'd give this a more specific name like `context`. It's not really the uninstall message, rather the context it's being uninstalled from.

```suggestion
  const context =
```